src/taipy/gui/_gui_cli.py

```diff
@@ -9,15 +9,15 @@
 # an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
 # specific language governing permissions and limitations under the License.
 
-from taipy.config.common._argparser import _Argparser
+from taipy._cli._base_cli import _CLI
 
 
 class _GuiCLI:
     """Command-line interface of GUI."""
 
     @classmethod
-    def _create_parser(cls):
-        gui_parser = _Argparser._add_groupparser("GUI", "Optional arguments for GUI service")
+    def create_parser(cls):
+        gui_parser = _CLI._add_groupparser("Taipy GUI", "Optional arguments for Taipy GUI service")
```

Comment:
What's the point of the 'Taipy ' prefix?

Reply:
Just that when we show the help message, it is clear that these options are for Taipy GUI